pkg/manager/member/utils.go

```diff
@@ -32,6 +32,8 @@ import (
 const (
 	// LastAppliedConfigAnnotation is annotation key of last applied configuration
 	LastAppliedConfigAnnotation = "pingcap.com/last-applied-configuration"
+	// LastAppliedPodTemplateAnnotation is annotation key of last applied configuration
+	LastAppliedPodTemplateAnnotation = "pingcap.com/last-applied-pod-template"
```

Comment:
It is possible to use `LastAppliedConfigAnnotation` because StatefuSetSpec contains PodTemplateSpec?

Reply:
Make sense

Reply:
Addressed in https://github.com/pingcap/tidb-operator/pull/1414/commits/498009cb1c95aa36c9ca3202d20612d02997f6dd